src/SymphonyOSS.RestApiClient/Generated/OpenApi/generate.bat

```diff
@@ -37,15 +37,13 @@ dotnet ..\..\..\..\tools\SwaggerOperationIdGenerator\SwaggerOperationIdGenerator
 dotnet ..\..\..\..\tools\SwaggerOperationIdGenerator\SwaggerOperationIdGenerator\bin\Debug\netcoreapp1.1\SwaggerOperationIdGenerator.dll temp\authenticatorAPI.json >temp\authenticatorAPI.operations.json
 dotnet ..\..\..\..\tools\SwaggerOperationIdGenerator\SwaggerOperationIdGenerator\bin\Debug\netcoreapp1.1\SwaggerOperationIdGenerator.dll temp\podAPI.json >temp\podAPI.operations.json
 
-
-
 rmdir /s /q AgentApi
 rmdir /s /q AuthenticatorApi
 rmdir /s /q PodApi
 
-call nswag run Config\agentAPI.nswag
-call nswag run Config\authenticatorAPI.nswag
-call nswag run Config\podAPI.nswag
+call ..\..\..\..\..\NSwagBinary\net461\NSwag.exe run Config\agentAPI.nswag
```

Comment:
We should revert these changes if the NSwag PR is accepted before we merge this one